app/partials/content.html

```diff
@@ -1,15 +1,77 @@
+<div class="content">
+
 <div class="row marketing">
-  <div class="col-lg-6">
-    <h4>HTML5 Boilerplate</h4>
-    <p>HTML5 Boilerplate is a professional front-end template for building fast, robust, and adaptable web apps or sites.</p>
 
-    <h4>Sass</h4>
-    <p>Sass is the most mature, stable, and powerful professional grade CSS extension language in the world.</p>
+  <div class="col-md-4 mark-shoe col-md-offset-1">
+    <div class="col-md-6 div-shoe">
```

Comment:
When using classes, don't tight the design with the content. Here's an example: you're using classes div-shoe, div-tshirt, div-short, the problem comes when the content of these pieces changes, so instead of a shoe, it now displays a watch, for example. Then that class will not make sense anymore.

Try to use more generic class names. And in this specific case, all of the 3 elements in this section, need to share the same style, they need to be scalable (this means, that we can put 50 more different products in there and it will still looks good) because that's what we're going to do when we add javascript, the product will by dynamic, and you'll not know which product is it, a shoe, a tshirt or a short.